riak_test/yz_solrq_test.erl

```diff
@@ -55,6 +80,16 @@ confirm() ->
     ok = create_indexed_bucket(PBConn, Cluster, ?BUCKET1, ?INDEX1),
     ok = create_indexed_bucket(PBConn, Cluster, ?BUCKET2, ?INDEX2),
     ok = create_indexed_bucket(PBConn, Cluster, ?BUCKET3, ?INDEX3),
+    ok = create_indexed_bucket(PBConn, Cluster, ?BUCKET4, ?INDEX4),
+    ok = create_indexed_bucket(PBConn, Cluster, ?BUCKET5, ?INDEX5),
+    ok = create_indexed_bucket(PBConn, Cluster, ?BUCKET6, ?INDEX6),
+    ok = create_indexed_bucket(PBConn, Cluster, ?BUCKET7, ?INDEX7),
+    ok = create_indexed_bucket(PBConn, Cluster, ?BUCKET8, ?INDEX8),
+    ok = create_indexed_bucket(PBConn, Cluster, ?BUCKET9, ?INDEX9),
+    ok = create_indexed_bucket(PBConn, Cluster, ?BUCKET10, ?INDEX10),
+    ok = create_indexed_bucket(PBConn, Cluster, ?BUCKET11, ?INDEX11),
+    ok = create_indexed_bucket(PBConn, Cluster, ?BUCKET12, ?INDEX12),
```

Comment:
In #621 I moved this `create_indexed_bucket` function into the `yz_rt` module so that it could be reused by other tests. We'll have to do some conflict resolution at some point.